packages/launchpad/src/setup/OpenBrowserList.vue

```diff
@@ -27,7 +27,7 @@
           }"
         >
           <Tooltip
-            v-if="!browser.isVersionSupported"
+            v-if="browser.warning"
```

Comment:
This also fixes an issue where the `chromeWebSecurity` warning did not display.

Reply:
I think you'd add it in this file potentially: https://github.com/cypress-io/cypress/blob/develop/packages/launchpad/cypress/e2e/config-files-error-handling.cy.ts#L96

Just pick a system-tests project that has `chromeWebSecurity: false` and it should show up.

Reply:
There are percy snapshots for the warning icon itself, does that cover the need for a test?